test/lib/sstable_test_env.hh

```diff
@@ -127,6 +127,14 @@ public:
         return reusable_sst(std::move(schema), _impl->dir.path().native(), generation);
     }
 
+    future<shared_sstable> reusable_sst(schema_ptr schema, shared_sstable sst) {
```

Comment:
I've came across a call to this overload above, but then noticed that there are many other places that call it and it wasn't obvious why. So my question is -- if the intention of this overload is to re-open an sstable from an existing one, what could be the reason for _not_ using the `sst`'s schema?